openedx/core/djangoapps/certificates/config/waffle.py

```diff
@@ -8,6 +8,7 @@
 WAFFLE_NAMESPACE = u'certificates'
 
 # Switches
+AUTO_CERTIFICATE_GENERATION_ENABLED = u'auto_certificate_generation_enabled'
```

Comment:
Extremely nitty, but could you get rid of the "enabled" in this name? After all, waffle switches have to be "activated", so semantically, "auto certificate generation" being "active" does describe the state of things correctly.